hammerdb/azuredeploy.parameters.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "numberOfWorkers": {
+            "value": 10
```

Comment:
do we want this to be the default?

Reply:
I guess we should set this to a minimum where we have meaningful results. e.g with a cluster of size 1, the comparision is not that important maybe.

Maybe 4 could be the default?